### PR TITLE
Add lua deg function to documentation

### LINKED ISSUE #3391: [Term Entry] Lua Math .deg()
### Reason/inspiration (optional)

We would like a new term entry on the `.deg()` method for Lua: [Math](https://www.codecademy.com/resources/docs/lua/mathematical-library). The entry should go in a new file under `docs/content/lua/concepts/mathematical-library/terms/deg/deg.md`.

The entry should describe what the method does. Be sure to include the following sections:

- A "Syntax" section with a ` ```pseudo` code snippet.
- An "Example" section with a ` ```lua `code snippet.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by laisvigas**
### Description

Closes #3391

The documentation provides comprehensive information about Lua's math.deg() function, explaining how it converts angles from radians to degrees. It covers the function's purpose, syntax, usage, and includes an illustrative example, ensuring a clear understanding of its implementation in Lua programming.

### Type of Change

- Adding a new entry
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by laisvigas** [comment]
I realized there were some unintended files included in the initial submission. However, I have resolved these issues, and I believe the pull request is now in good shape. Please take a moment to go through the changes and provide any feedback or suggestions.

**Comment by laisvigas** [comment]
I realized there were some unintended files included in the initial submission. However, I have resolved these issues, and I believe the pull request is now in good shape. Please take a moment to go through the changes and provide any feedback or suggestions.

File: content/lua/concepts/mathematical-library/terms/deg/deg.md
Comment (dakshdeepHERE) [inline content/lua/concepts/mathematical-library/terms/deg/deg.md:2]:
```suggestion
Title: ‘.deg()’
```

File: content/lua/concepts/mathematical-library/terms/deg/deg.md
Comment (dakshdeepHERE) [inline content/lua/concepts/mathematical-library/terms/deg/deg.md:19]:
```suggestion
> **Note**: It's important to be cautious of rounding errors when converting between degrees and radians due to computer's floating-point representation, especially for small or large angles. Rounding errors occur when converting between degrees and radians due to the inherent limitations of computer representation of numbers.
```

File: content/lua/concepts/mathematical-library/terms/deg/deg.md
Comment (dakshdeepHERE) [inline content/lua/concepts/mathematical-library/terms/deg/deg.md:43]:
```suggestion
local angleInRadians = math.pi / 4

local angleInDegrees = math.deg(angleInRadians)

print("Angle of the right triangle in degrees: " .. angleInDegrees)
```

**Review by dakshdeepHERE — CHANGES_REQUESTED** [review]
Few changes from my side. Otherwise this looks good

**Comment by dakshdeepHERE** [comment]
Few changes from my side @laisvigas.

**Comment by laisvigas** [comment]
> @laisvigas the other files and the commit came here can be due to two issue:
> 
> * this branch `add-lua-deg-function-to-documentation` is a sub branch/child branch of some other branch. To confirm it run this locally in CLI
> 
> ```shell
> git log --oneline --graph --decorate --all --simplify-by-decoration
> ```
> 
>  * Or you committed those commit on your main branch. So basically those commit will appear everywhere in all other branches that you will create from now on. If this is the case then I will recommend rebasing the branch.

@dakshdeepHERE Thank you for your guidance. GIT and GitHub are still relatively new concepts for me. Thanks to Hacktoberfest, I've been diving deep into the world of version control and collaboration, and I'm learning a lot in the process. I used the command you suggested: `git log --oneline --graph --decorate --all --simplify-by-decoration`, and here's what I found:

Screenshot 2023-10-27 at 12 20 52

If there are any specific actions you'd like me to take based on this information, please let me know. Otherwise, should I wait for the second review before making any additional changes?

**Review by dakshdeepHERE — APPROVED** [review]

**Comment by dakshdeepHERE** [comment]
Based on the Git log you provided, it appears that the branch `add-lua-deg-function-to-documentation` (HEAD) has a commit history that shows it was branched off from `origin/add-lua-deg-function-to-documentation`. This is evident from the branching pattern in the Git log, where there's a merge commit from `origin/add-lua-deg-function-to-documentation` into your current branch.

The line in your Git log indicates this branching:

```
* 23cc9ec9 (HEAD - add-lua-deg-function-to-documentation, origin/add-lua-deg-function-to-documentati on) Add Lua math.deg() function to the documentation and exclude wrong files
| * 6f3f7de6 (origin/add-asin-method-to-Lua-documentation) Merge branch 'main' into add-asin-method-t o-Lua-documentation
```

In this log, you can see that `23cc9ec9` (your current branch) is a child branch of `origin/add-lua-deg-function-to-documentation`. The merge commit `6f3f7de6` indicates that changes from `origin/add-asin-method-to-Lua-documentation` were merged into your branch.

So, based on this information, your branch `add-lua-deg-function-to-documentation` is indeed a sub-branch of `origin/add-lua-deg-function-to-documentation`.

**Comment by dakshdeepHERE** [comment]
Also the entry looks good for 2nd review.

**Review by caupolicandiaz — APPROVED** [review]
@laisvigas Just made a few direct edits to correct some formatting and simplify.

**Pull request merged**
